CHANGELOG.next.toml

```diff
@@ -27,4 +27,16 @@ author = "Velfi"
 message = "Enable aws-smithy-runtime to compile in rustc 1.72.1"
 references = ["smithy-rs#3675"]
 meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client" }
-author = "dastrom"
\ No newline at end of file
+author = "dastrom"
+
+[[aws-sdk-rust]]
+message = "Fix S3 ListParts API paginator infinite loop."
+references = ["aws-sdk-rust#1143"]
+meta = { "breaking" = false, "tada" = false, "bug" = true }
+author = "landonxjames"
+
+[[smithy-rs]]
+message = "Fix S3 ListParts API paginator infinite loop."
+references = ["aws-sdk-rust#1143"]
+meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client"}
+author = "landonxjames"
```

Comment:
Nit: since this work is specific to AWS SDK for Rust, I think this entry for `smithy-rs` (i.e. a generic SDK code-generated from smithy) should be removed.

Reply:
Removed in https://github.com/smithy-lang/smithy-rs/pull/3679/commits/a14a2eacf112eae221c3ca3abc47b34af6f360bb